modules/file-integrity-operator-exploring-daemon-sets.adoc

```diff
@@ -12,27 +12,31 @@ To find the daemon set that represents a `FileIntegrity` object, run:
 
 [source,terminal]
 ----
-$ oc get ds/aide-ds-$file-integrity-object-name
+$ oc -n openshift-file-integrity get ds/aide-ds-example-fileintegrity-ip-10-0-139-137.us-east-2.compute.internal
```

Comment:
the ds name should be the same with fileintegrity object name. To align with line 22, need to updated to:
$ oc get ds/aide-worker-fileintegrity
```
$ oc get ds
NAME                         DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
aide-example-fileintegrity   6         6         6       6            6           <none>          5m16s
$ oc get pod --show-labels 
NAME                                       READY   STATUS    RESTARTS      AGE     LABELS
aide-example-fileintegrity-2zlnk           1/1     Running   0             6m31s   app=aide-example-fileintegrity,controller-revision-hash=8687477b94,file-integrity.openshift.io/owner=example-fileintegrity,file-integrity.openshift.io/pod=,pod-template-generation=1
aide-example-fileintegrity-55lz2           1/1     Running   0             6m31s   app=aide-example-fileintegrity,controller-revision-hash=8687477b94,file-integrity.openshift.io/owner=example-fileintegrity,file-integrity.openshift.io/pod=,pod-template-generation=1
aide-example-fileintegrity-bjql7           1/1     Running   0             6m31s   app=aide-example-fileintegrity,controller-revision-hash=8687477b94,file-integrity.openshift.io/owner=example-fileintegrity,file-integrity.openshift.io/pod=,pod-template-generation=1
aide-example-fileintegrity-dfg98           1/1     Running   0             6m31s   app=aide-example-fileintegrity,controller-revision-hash=8687477b94,file-integrity.openshift.io/owner=example-fileintegrity,file-integrity.openshift.io/pod=,pod-template-generation=1
aide-example-fileintegrity-pwgfq           1/1     Running   0             6m31s   app=aide-example-fileintegrity,controller-revision-hash=8687477b94,file-integrity.openshift.io/owner=example-fileintegrity,file-integrity.openshift.io/pod=,pod-template-generation=1
aide-example-fileintegrity-sfnj9           1/1     Running   0             6m31s   app=aide-example-fileintegrity,controller-revision-hash=8687477b94,file-integrity.openshift.io/owner=example-fileintegrity,file-integrity.openshift.io/pod=,pod-template-generation=1
file-integrity-operator-6cd76cdc65-x4492   1/1     Running   1 (11m ago)   11m     name=file-integrity-operator,pod-template-hash=6cd76cdc65
```